extensions/Worker.Extensions.EventHubs/release_notes.md

```diff
@@ -4,6 +4,6 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 6.3.1
+### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 6.4.0
```

Comment:
This can be a patch change and kept in sync with the event hubs version.
```suggestion
### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 6.3.3
```